docs/reference/modules.rst

```diff
@@ -13,14 +13,19 @@ Core Functionality
 
    binning
    constants
-   integrate
-   interpolate
-   ndimage
-   optimize
-   signal
    spatial
    units
 
+Wrappers
+--------
```

Comment:
```suggestion
SciPy Wrappers
--------------
```
?

Reply:
The section just links to `scipy`, the target of the link are the `SciPy Wrappers`, not the whole section.